VitoConnect/compute_name.php

```diff
@@ -1,48 +1,84 @@
+    "heating.circuits.%d.operating.programs.standby" => "Heizkreis %d: Heizung Standby (AUS)",
+    "heating.circuits.%d.sensors.temperature.supply" => "Heizkreis %d: Kesseltemperatur",
     "heating.circuits.%d" => "Heizkreis %d",
-    "heating.burner.statistics" => "",
-    "heating.burner.modulation" => "Modulation",
-    "heating.burner" => "Brennerstatus",
-    "heating.boiler.serial" => "Meine Anlage: Seriennummer",
-    "heating.boiler.sensors.temperature.main" => "Kesseltemperatur"
+    "heating.compressor.%d.statistics" => "Heizkreis %d: Verdichter",
+    "heating.compressor.%d" => "Heizkreis %d: Betrieb Verdichter",
+    "heating.compressor.statistics" => "Verdichter",
+    "heating.compressor" => "Betrieb Verdichter",
+    "heating.configuration.multiFamilyHouse" => "Mehrfamilienhaus",
+    "heating.dhw.charging" => "Warmwasser Aufladung",
+    "heating.dhw.oneTimeCharge" => "Warmwasser Einmalaufladung",
+    "heating.dhw.pumps.circulation.schedule" => "Warmwasser Zirkulationspumpe Zeitprogramm",
+    "heating.dhw.schedule" => "Warmwasser Zeitprogramm",
+    "heating.dhw.sensors.temperature.hotWaterStorage.top" => "Warmwassertemperatur",
+    "heating.dhw.sensors.temperature.hotWaterStorage" => "Warmwasser Temperatur",
+    "heating.dhw.temperature.hysteresis" => "Warmwassertemperatur Hysterese",
+    "heating.dhw.temperature.main" => "Warmwasser Primäre Wunschtemperatur",
+    "heating.dhw.temperature.temp2" => "Warmwassertemperatur 2 Soll",
+    "heating.dhw.temperature" => "Warmwasser Wunschtemperatur",
+    "heating.dhw" => "Warmwasser",
+    "heating.gas.consumption.dhw" => "Gasverbrauch Warmwasserbereitung",
+    "heating.gas.consumption.heating" => "Gasverbrauch Raumbeheizung",
+    "heating.power.consumption" => "Energieverbrauch",
+    "heating.sensors.temperature.outside" => "Außentemperatur",
+    "heating.sensors.temperature.return" => "Rücklauftemperatur Sekundärkreis",
+    "ventilation.operating.modes.active" => "Lüftung Betriebsart",
+    "ventilation.operating.modes.standard" => "Lüftung Grundbetrieb",
+    "ventilation.operating.modes.standby" => "Lüftung Standby (AUS)",
+    "ventilation.operating.modes.ventilation" => "Lüftungsautomatik",
+    "ventilation.operating.programs.active" => "Lüftung Betriebsart",
+    "ventilation.operating.programs.basic" => "Lüftung MIN Betrieb",
+    "ventilation.operating.programs.intensive" => "Lüftung intensiever Betrieb",
+    "ventilation.operating.programs.reduced" => "Lüftung reduzierter Betrieb",
+    "ventilation.operating.programs.standard" => "Lüftung normaler Betrieb",
+    "ventilation.operating.programs.standby" => "Lüftung Standby (AUS)",
+    "ventilation.schedule" => "Lüftung Zeitprogramm",
+    "ventilation" => "Betrieb Lüftung"
+
 ]);
 
 define("READABLE_NAMES", [
     "value" => "",
     "active" => "",
-    "hours" => "Brennerbetriebsstunden",
-    "starts" => "Brennerstarts",
+    "hours" => "Betriebsstunden",
```

Comment:
Gerne analog zu anderen Werten in Klammern setzen, denke auch an das Leerzeichen